README.md

```diff
@@ -11,13 +14,14 @@
 
 ## Overview
 
-The Starling Integrity Backend ingests ZIPs from the filesystem and operates on them for archival and preservation in various ways.
+The Starling Integrity Backend ingests data bundles from the filesystem and operates on them for authenticated archival.
```

Comment:
Should tie in the word "input bundle" here since this is what we are using throughout?